apis/core/lib/pipeline/trigger.ts

```diff
@@ -33,9 +33,19 @@ export const local = trigger(job => {
   }
 })
 
-export async function gitlab(job: NamedNode): Promise<void> {
-  throw new Error('Not implemented')
-}
+export const gitlab = trigger(job => {
+  const form = new URLSearchParams({
+    token: env.PIPELINE_TOKEN,
+    ref: env.maybe.PIPELINE_BRANCH || 'master',
```

Comment:
I don't think we want this here. There will probably be only one branch on GitLab
```suggestion
    ref: 'master',
```

Reply:
If you think so

Reply:
On GitHub the container executes directly from sources. Thus, having the branch option is good to be able to run different code version.

GitLab runs from pre-built containers. So either we could make the branch a trigger variable. Otherwise, if the job start command changes, we can differentiate that in the gitlab yaml without creating branches